### DISCUSSION TITLE
Next release schedule

### DISCUSSION THREAD

**Discussion by MKergall** [osmdroid/osmdroid#1801] [General]
Dear Osmdroid contributors, 
Last version is nearly one year ago. Various fixes have been done. 
What do you think about releasing a 6.1.12 soon? 

(https://github.com/osmdroid/osmdroid/pull/1715 to merge before?)

**Comment by monsieurtanuki** [comment]
Hi @MKergall! I agree, merging #1715 before.

**Reply by spyhunter99** [reply]
no problem, ping me we're ready

**Reply by monsieurtanuki** [reply]
@spyhunter99 I've just merged #1715. I think we're ready.

**Comment by spyhunter99** [comment]
on it now

**Comment by guillaumecardon** [comment]
Hello,

The 6.1.18 version have the feature "OneFinger Zoom". Android Studio cannot get the "master" branch containing the fix without a version
`    implementation ("org.osmdroid:osmdroid-android:master")`
When do you plan to deploy the next version ?
Thanks for your great work !

**Comment by spyhunter99** [comment]
i'll crank it out today
